### ISSUE TITLE
v9-gui: Can't set a bandwidth balance to a portal module

### ISSUE THREAD

**Issue by cgx** [inverse-inc/packetfence#4494]
Adding a bandwidth balance to the actions of a portal module can't be saved.

**Comment by jrouzierinverse** [comment]
It seems to work on my side.

```bash
ID="default_login_policy_$$"
/usr/local/pf/sbin/pfperl-api get -M POST  /api/v1/config/portal_modules <<ITEM
{
  "actions": [{"type":"set_bandwidth_balance", "arguments" :"2323"  }],
  "aup_template": "aup_text.html",
  "description": "Username/password login",
  "id": "$ID",
  "multi_source_ids": null,
  "not_deletable": true,
  "not_sortable": false,
  "pid_field": "username",
  "signup_template": "signin.html",
  "type": "Authentication::Login",
  "with_aup": 1
}
ITEM

echo

/usr/local/pf/sbin/pfperl-api get "/api/v1/config/portal_module/$ID" | jq

```

Please check in the front end what you are sending

**Comment by cgx** [comment]
We're sending *type* and *value*. Why *arguments*? What do you think @satkunas since you worked on `pfFormPrefixMultiplier`?

**Comment by jrouzierinverse** [comment]
That is the name it has always been.

**Comment by satkunas** [comment]
@cgx I'll need to produce a pfFieldTypeArguements.vue component to use `arguements` field.

@jrouzierinverse will modify the backend to accept `value` instead, so we can get away with using existing naming.

**Comment by jrouzierinverse** [comment]
fixed by a41363df45ee7b88bfc6a3fe9e80aa4c899f77d3
